.buildkite/.envrc

```diff
@@ -1 +1 @@
-eval $(lorri direnv --shell-file default.nix)
```

Comment:
"use nix" is a valid .envrc? 🤔

Reply:
Yes it is: https://direnv.net/man/direnv-stdlib.1.html#codeuse-ltprogramnamegt-ltversiongtcode
I found nix-direnv which is better than lorri.